drivers/dma/dma_pl330.c

```diff
@@ -582,7 +582,7 @@ static const struct dma_driver_api pl330_driver_api = {
 	.stop = dma_pl330_transfer_stop,
 };
 
-static const struct device DEVICE_NAME_GET(dma_pl330);
+static const struct device DEVICE_DT_NAME_GET(DT_DRV_INST(0));
```

Comment:
I believe this should be replaced by `DEVICE_DT_DECLARE(0)`.  As implemented I *think* it would make the device unreferencable by `DEVICE_DT_GET()` outside this module (as normal device declare provides unqualified visibility).

Reply:
Changed made to `DEVICE_DT_DECLARE(0)`